mpas_analysis/ocean/meridional_overturning_circulation.py

```diff
@@ -400,65 +426,68 @@ def _compute_moc_postprocess(config, streams, variableMap, calendar,
-            # create dimension
-            ncFile.createDimension('Time', 0)
-            # create variables
-            Timevar = ncFile.createVariable('Time', 'f4', ('Time',))
-            Timevar.description = 'days since start of simulation'
-            mocAtlantic26var = ncFile.createVariable('mocRapidArrayMax',
-                                                     'f4', ('Time',))
-            mocAtlantic26var.description = 'Max MOC Atlantic streamfunction'\
-                                           ' nearest to RAPID Array latitude'\
-                                           ' (26.5N)'
-            mocAtlantic26var.units = 'Sv (10^6 m^3/s)'
-            # save variables
-            Timevar[:] = Time[ncount-12:ncount]
-            mocAtlantic26var[:] = mocAtlantic26[ncount-12:ncount]
-            ncFile.close()
-        else:
-            ncount += 12
-            # Read from file
-            print '     Read previously computed Atlantic MOC at 26.5N'\
-                  ' from file...'
-            ncFile = netCDF4.Dataset(outputFileTseries, mode='r')
-            Time[ncount-12:ncount] = ncFile.variables['Time'][:]
-            mocAtlantic26[ncount-12:ncount] = ncFile.variables['mocRapidArrayMax'][:]
-            ncFile.close()
-
-    # Store max Atlantic MOC at 26.5 to dictionary
-    mocDictTseries = {'Time': {'dims': ('Time'), 'data': Time},
-                      'mocRapidArrayMax': {'dims': ('Time'),
-                                           'data': mocAtlantic26}}
-
-    return (mocDictClimo, mocDictTseries)
+    outputFileTseries = '{}/mocTimeSeries.nc'.format(outputDirectory)
+
+    continueOutput = os.path.exists(outputFileTseries)
+    if continueOutput:
+        # read in what we have so far
+        dsMOCTimeSeries = xr.open_dataset(outputFileTseries)
+        # force loading and then close so we can overwrite the file later
+        dsMOCTimeSeries.load()
+        dsMOCTimeSeries.close()
+    else:
+        # create a new data set from a dictionary
+        dictonary = {
+            'Time': {'dims': ('Time'),
+                     'units': 'days since 0001-01-01',
+                     'data': []},
+            'mocAtlantic26': {'dims': ('Time'),
+                              'units': 'Sv (10^6 m^3/s)',
+                              'description': 'Max MOC Atlantic streamfunction '
+                                             'nearest to RAPID Array '
+                                             'latitude (26.5N)',
+                              'data': []}}
+        dsMOCTimeSeries = xr.Dataset.from_dict(dictonary)
+
+    newEntries = False
+    for tIndex in range(ds.Time.size):
+        if ds.Time[tIndex].values in dsMOCTimeSeries.Time.values:
+            # we have it already, so no need to recompute
+            continue
+        date = days_to_datetime(ds.Time[tIndex], calendar=calendar)
+        print '   date: {:04d}-{:02d}'.format(date.year, date.month)
```

Comment:
could you add 1-2 trailing spaces before `date`? (and this should be the last trivial comment)